### ISSUE TITLE
JS select() not working in Chrome or Chromium

### ISSUE THREAD

**Issue by scrow** [scrow/wx4akq-offline-tools#7]
After showing or hiding the criteria DIV, expected behavior is that the eventDate field will be select()ed, and this is not working in Chromium.

**Comment by scrow** [comment]
No workarounds found...

**Comment by scrow** [comment]
Appears to be a bug in the browser.  No known fix.  Leaving this open for a while.

**Comment by scrow** [comment]
Fixed.  The issue is that only Opera and Chrome support `input[type=date]` and `input[type=time]` fields, and those fields do not support `.select()`.  Since Firefox, Safari, and Internet Explorer don't support those field types, they're treating them as regular text input fields, which DO support `.select()`.

What I have found is that the date and time field types support `.focus()`, which is good enough for us.  So what I've done is turned each `.select()` call that references a date or time field into a focus and then a `.select()`.  This way, at a minimum we will see the desired `.focus()` behavior, and those browsers which lack support for date and time fields will then do a `.select()`:

```
document.getElementById('eventTime').focus();  /* for Chrome, Opera */
document.getElementById('eventTime').select();  /* for all other browsers */
```

Issue considered resolved.
